crates/api_structs/src/calendar/dtos.rs

```diff
@@ -9,10 +9,20 @@ use ts_rs::TS;
 pub struct CalendarDTO {
     /// UUID of the calendar
     pub id: ID,
+
     /// UUID of the user that owns the calendar
     pub user_id: ID,
+
+    // Name of the calendar (optional)
+    pub name: Option<String>,
+
+    // Key of the calendar (optional)
+    // When defined, this is unique per user
```

Comment:
I think these should be `///` or else the generated JS doesn't carry the docs through.

Reply:
Oh, good catch, that's indeed the problem 😅